# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/objc_video_capture/video_capture.h
# Commit: 703360f0b861
# Full Hash: 703360f0b8611916dfb3724eda7f35345d3f4be7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2022-12-13 04:11:09
# Regressor Bug: 1451394
# File Overlap Count: 1
# Description:
#   Bug 1451394 - Integrate with the libwebrtc camera backend for Mac. r=webrtc-reviewers,jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D163684
# ==============================================================================

diff -r 49002523b70d -r 703360f0b861 dom/media/systemservices/objc_video_capture/video_capture.h
--- a/dom/media/systemservices/objc_video_capture/video_capture.h	Mon Dec 12 15:47:22 2022 +0000
+++ b/dom/media/systemservices/objc_video_capture/video_capture.h	Mon Dec 12 15:47:22 2022 +0000
@@ -16,8 +16,7 @@
 
 @class RTCVideoCaptureIosObjC;
 
-namespace webrtc {
-namespace videocapturemodule {
+namespace webrtc::videocapturemodule {
 class VideoCaptureIos : public VideoCaptureImpl {
  public:
   VideoCaptureIos();
@@ -37,7 +36,6 @@
   VideoCaptureCapability capability_;
 };
 
-}  // namespace videocapturemodule
-}  // namespace webrtc
+}  // namespace webrtc::videocapturemodule
 
 #endif  // MODULES_VIDEO_CAPTURE_OBJC_VIDEO_CAPTURE_H_